cip/CIP2016-07-07-Parameter-syntax.adoc

```diff
@@ -0,0 +1,91 @@
+= CIP2016-07-07 - Parameter syntax
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+This CIP suggests changing syntax for parameter usage, from curly braces (`{}`) to dollars (`$`).
+Curly brackets are used by the language for literal maps and property predicates, and this overloading may cause confusion in usage.
+Using a different, unique character causes parameters to be more visible for the human eye, and by using only a single character for the parameter instead of one opening and one closing, the syntax becomes somewhat cleaner.
```

Comment:
I would suggest something along these lines: 
'In order to make parameters more readable, a single, unique and hitherto unused character'
'A single, unique and hitherto unused character used as a prefix increases the readability of parameters when compared with delimiting parameters using overloaded characters. Moreover, this allows for cleaner syntax.'